src/utils/getValue.ts

```diff
@@ -0,0 +1,14 @@
+import { Accessor } from './accessor';
+import { path } from './path';
+
+export function getValue<D, C>(value: D, accessor: ''): null;
```

Comment:
Can we change `''` to something else? maybe `null`?

Reply:
Maybe no. I am not sure.